tests/unit/plugins/modules/cloud/misc/test_terraform.py

```diff
@@ -1,12 +1,188 @@
-# Copyright: (c) 2019, Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
```

Comment:
This file was modified in another PR (some common boilerplate code was added to many different files in there), you need to rebase your PR.

Reply:
sure. will do that